barretenberg/cpp/src/barretenberg/ecc/curves/bn254/fq.hpp

```diff
@@ -66,6 +66,21 @@ class Bn254FqParams {
 
     static constexpr uint64_t r_inv = 0x87d20782e4866389UL;
 
+    static constexpr uint64_t r_inv_0 = 0x327d7c1b18f7bd41UL;
```

Comment:
I'm sure you wanted to maintain the absolutely pristine absence of comments in this file but would you mind adding some anyway :). E.g. a note about the significance of `r`. Also, it seems like many of these constants were computed using the python script included in this PR? If so can you make a note of that in the comments?